.github/workflows/main.yml

```diff
@@ -388,6 +388,24 @@ jobs:
             build-type: Debug
             arch: x64
             toolchain: ""
+          - os: windows-2022
+            build-type: Release
+            arch: ARM64
+            toolchain: ""
+          - os: windows-2022
+            build-type: Debug
+            arch: ARM64
```

Comment:
I'm assuming the tests will fail to run. We need a `build-only` option in the matrix.  There is one above
https://github.com/microsoft/snmalloc/blob/b8e9e99cf096357d37c67d423ccb657652879ba1/.github/workflows/main.yml#L89

Reply:
Thanks, that should be fixed in the latest commit. I addressed this on my latest commit. It looks like you have to manually allow the workflows to run again. I expect them to pass now.